examples/browser/src/main.rs

```diff
@@ -51,18 +51,15 @@ fn main() -> zeroconf::Result<()> {
 
     loop {
         // calling `poll()` will keep this browser alive
-        event_loop.poll(Duration::from_secs(0))?;
+        event_loop.poll(Duration::from_secs(1))?;
```

Comment:
Why?

Reply:
Thanks for looking.

> Why?

When I first tried the example, there was no output and my CPU fans fired up to 100%. So my first thought was that the example was broken. Only once I started looking into the code I realized what was going on.

This was on Linux using Avahi where the `avahi_simple_poll_iterate` does not block until there is an event on the socket when the sleep time is zero. I'm not sure about Bonjour, but I think the situation is similar?

Unless I'm misunderstanding something, having a longer sleep duration has no drawbacks because if there are any events before the sleep time expires, the `poll` function breaks before that and the events are still handled "as fast as possible". And it has the benefit of not frying the CPU.